src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import 'jest-styled-components';
+import { cleanup, render, fireEvent } from '@testing-library/react';
+
+import { Collapsible } from '..';
+import { Box } from '../../Box';
+import { Button } from '../../Button';
+import { Grommet } from '../../Grommet';
+import { Text } from '../../Text';
+
+describe('Collapsible', () => {
+  afterEach(cleanup);
+
+  test('open', () => {
+    const { container } = render(
+      <Grommet>
+        <Collapsible open>
+          <Box background="dark-1">Box Body</Box>
```

Comment:
Is setting the background prop necessary for this test, the test is called open so I am thinking it is just testing that the open prop can be added to collapsible and the box background doesn't matter?